config/nunjucks/globals.js

```diff
@@ -15,6 +15,7 @@ const {
   routes: toolsRoutes,
 } = require('../../app/tools')
 const i18n = require('../i18n')
+const { AUTH_BASE_URL } = require('../index.js')
```

Comment:
This file is already included on line 3 so you can add a definition to that instead of including it again.
```suggestion
```